### ISSUE TITLE
"Skip to main content" link is not provided in the page, because of this keyboard user will have to navigate through all repeating contents to reach the main content : A11y_WCP URLs - ONNX Runtime_Home_Home_BypassBlocks

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20151]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; A11yWCAG2.1; #A11yMAS; #BypassBlocks; #WCAG2.4.1;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2
### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Now check that "Skip to main content" link is not provided in the page, because of this keyboard user will have to navigate through all repeating contents to reach the main content.

### Actual Results:
"Skip to main content" link is not provided in the page, because of this keyboard user will have to navigate through all repeating contents to reach the main content.

### Expected Results:
"Skip to main content" link should be provided in the page with proper functionality.

### User Impact:
Keyboard user will face difficulties in reaching the main content of the page.

### Attachment:

[skip to main content.zip](https://github.com/microsoft/onnxruntime/files/14811001/skip.to.main.content.zip)

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URL:  https://onnxruntime.ai/
Edge version:125.0.2535.92 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
For reference attachment is added.

#Regressed:06-17-24; #Closed;

### Attachment:

[Recording-20240617_110852.webm](https://github.com/microsoft/onnxruntime/assets/142479541/f521ed5b-1763-4f7f-a28e-fb20cfee7ced)
